examples/optimize_rrt_path.py

```diff
@@ -116,16 +116,17 @@ def random_valid_state():
             traj_gen = traj.generate(dt)
             q_vec = traj_gen[1]
 
-            # Display the trajectory and points along the path.
-            plt.ion()
-            traj.visualize(
-                dt=dt,
-                joint_names=model.names[1:],
-                show_position=True,
-                show_velocity=True,
-                show_acceleration=True,
-                show_jerk=True,
-            )
+            # plot the trajectory and points along the path.
+            if False:
```

Comment:
Putting an `if False` seems a bit strange. Maybe just make a boolean variable?

Reply:
I assume you meant to introduce a variable and directly use it, similar to how you do it in other places.